### ISSUE TITLE
JackettTray.exe flagged as Trojan by Windows Defender

### ISSUE THREAD

**Issue by avggeek** [Jackett/Jackett#2729]
Windows Defender on Windows 10 now seems to be flagging JackettTray.exe as a Trojan. Here is the error message that I'm seeing:

```
Trojan:Win32/Critet.BS
file: C:\ProgramData\Jackett\JackettTray.exe
```

Windows Defender reports definitions are last updated as of March 17, 2018.

This seems to be a problem with the latest definition update as it's affecting [other software](https://github.com/processing/processing/issues/5442) as well, including apparently KSP.

**Jackett version**:  0.8.763

**Comment by joelteixeira** [comment]
Same issue here but including console and service executables, registry keys and even the uninstall scripts. Is there any official communications regarding this?

**Comment by flightlevel** [comment]
This is a false positive, its an issue with Windows Defender

No major changes have been made to the code recently and Jackett code is 100% open source, you can go through it line by line yourself if you want

Search for 'windows defender Win32/Critet.BS' in Google and filter results to the last week. There is many reports of false positives

Your options are
1. Use another antivirus
2. Add Jackett folder to the exceptions in Defender
3. Wait for Microsoft to update their definitions
4. Stop using Jackett

**Comment by avggeek** [comment]
@flightlevel Agreed that this is a false positive - I even looked at the recent commits to JackettTray.exe to confirm that nothing is hinky. To work around the problem, I have excluded the exe from Windows Defender.

However, I filed the issue for 2 reasons - 1) to provide a reference for other folks who might get this error and start searching and 2) To check whether the developers would be willing to consider signing the app - my searching seems to suggest that signed apps are not as affected by this issue (I could be wrong).
